sql_generators/active_users/templates/fenix_checks.sql

```diff
@@ -27,9 +27,9 @@ distinct_client_count_nightly_base AS (
   WHERE
     DATE(submission_timestamp) = @submission_date
     AND mozfun.norm.fenix_app_info("org_mozilla_fenix", client_info.app_build).channel = "nightly"
-    -- NOTE: the below two tables are marked as depricated inside the GLEAN dictionary
-    -- however, they are still considered when generating active_users_aggregates metrics
-    -- this is why they are being considered here.
+    -- NOTE: the below two tables are marked as deprecated inside the GLEAN dictionary.
```

Comment:
@lucia-vargas-a which two tables are deprecated? I was confused coz the only table I saw below was the `client_info.client_id`

Reply:
Tables org_mozilla_fenix_nightly_live.baseline_v1 &
org_mozilla_fennec_aurora_live.baseline_v1.  @kik-kik could you confirm, since you found this while building the checks?

Reply:
yes, so if you look at GLEAN dictionary you can see that some of the Fenix tables are marked as deprecated, however, it looks like they were still being used in our models / table (and still get a small amount of data). I'm pretty sure I've mentioned this before to you @gkabbz .

https://dictionary.telemetry.mozilla.org/apps/fenix?itemType=app_ids&page=1

I could swear there were apps marked as deprecated in the dictionary when I looked, but don't see it anymore. However, if you look at `org.mozilla.fennec.aurora` specifically the description says `Nightly channel of Firefox for Android users migrated to Fenix; delisted in June 2020.` So my understanding would be that this specific app was deprecated in June 2020. Of course, delisted does not mean everyone suddenly stopped using it so data could still be flowing in.

But found it:
https://web.archive.org/web/20230330013651/https://dictionary.telemetry.mozilla.org/apps/fenix?itemType=app_ids&page=1

So to be honest, I'm not sure now whether these app ids are indeed deprecated or not. I linked both of you a conversation on Slack where I try to get some insight.


**In short: I'm not sure if this comment is valid.**

Reply:
Based on Jan-Erik's response, those two tables are still deprecated.

Reply:
So, perhaps that's sort of true. The deprecation label has been removed in October via:
https://github.com/mozilla/probe-scraper/pull/640/files

Reply:
Thanks @kik-kik, I updated the comment to reflect the actual status. For both tables the data continues to be reported and collected.